pkg/master/thirdparty/thirdparty.go

```diff
@@ -284,6 +284,7 @@ func (m *ThirdPartyResourceServer) InstallThirdPartyResource(rsrc *extensions.Th
 	m.genericAPIServer.HandlerContainer.Add(genericapi.NewGroupWebService(api.Codecs, path, apiGroup))
 
 	m.addThirdPartyResourceStorage(path, plural.Resource, thirdparty.Storage[plural.Resource].(*thirdpartyresourcedataetcd.REST), apiGroup)
+	registered.AddThirdPartyAPIGroupVersions(schema.GroupVersion{Group: group, Version: rsrc.Versions[0].Name})
```

Comment:
it won't, I add this because there are some calls in the apiserver which did the same check as above, for example: https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/extensions/thirdpartyresourcedata/codec.go#L373.

but yeah, I did two things here, and I am still doing various of validation.